### ISSUE TITLE
please add tag for 1.0.0 release

### ISSUE THREAD

**Issue by pravi** [aseemk/requireDir#55]
0.3.2 is the latest tag but npmjs.com has 1.0.0

**Comment by contra** [comment]
Tags or releases? What are you relying on these for?

**Comment by contra** [comment]
I've published them - https://github.com/aseemk/requireDir/releases/tag/1.0.0

Still curious what you use them for, though

**Comment by pravi** [comment]
We prefer to use github tarballs when creating deb packages https://tracker.debian.org/pkg/node-require-dir (to include tests and ensure no prebuilt files are present, which is not the case for most npm dist tarballs).

**Comment by contra** [comment]
@pravi Why manage npm modules on the debian package manager?

**Comment by pravi** [comment]
We package only those modules required for applications like gitlab or diaspora. We want `apt install gitlab` to setup a gitlab instance with all code complying with Debian policy (every generated code ie, transpiled, compiled from coffee/ts/ls etc should be built from source using tools available in debian). We are not targeting node/js developers, but system administrators who want a uniform method to manage all the software they use. require-dir is a build dependency of neo-async, which is a dependency of compression-webpack-plugin, which in turn dependency of gitlab.

**Comment by contra** [comment]
You have to turn an entire npm dependency tree into individual debian packages on a regular basis and keep them in sync/working/updated? Pouring one out for ya @pravi, that's rough - best of luck.

**Comment by pravi** [comment]
Yes, I have been on it for last two years. Major version bumps are problematic sometimes, but so far I got good support from node community when I needed code level changes. For example, I had to use babel 6 and webpack 3 for building axios where as they were still using babel 5 and webpack 1. See https://github.com/axios/axios/pull/1215 I have also cases like react which I have not been able to fix yet https://github.com/facebook/react/issues/12076 When we moved from chai 3 to 4, we had to fix around 6 modules. We try to keep only one version of a module or tool in the archive (as a last resort, we can still keep multiple versions).

I have been trying to get more people involved in maintaining these node modules, with some limited success. As I know it is not easy to do it alone. So I regularly interact with students and mentors them in packaging these node modules. But the number of people who stick around with me is very very small compared to the number of people I mentor.
